package/MDAnalysis/analysis/hole.py

```diff
@@ -1275,15 +1271,23 @@ def _process_orderparameters(self, data):
             raise ValueError(errmsg)
         return q
 
-    def run(self, **kwargs):
+    def run(self, start=None, stop=None, step=None):
         """Run HOLE on the whole trajectory and collect profiles.
 
-        Keyword arguments `start`, `stop`, and `step` can be used to only
-        analyse part of the trajectory.
+        Parameters
+        ----------
+        start : int, optional
+            First frame of trajectory to analyse, Default: None (first frame)
+        stop : int, optional
+            Last frame of trajectory to analyse, Default: None (last frame)
+        step : int, optional
+            Step between frames to analyse, Default: None (use every frame)
         """
-        start = kwargs.pop('start', self.start)
-        stop = kwargs.pop('stop', self.stop)
-        step = kwargs.pop('step', self.step)
+        start, stop, step = self.universe.trajectory.check_slice_indices(
+            start,
+            stop,
+            step
+        )
         hole_kw = self.hole_kwargs.copy()
         hole_kw.update(kwargs)
```

Comment:
I am pretty sure that this line needs to be removed as you don't have `kwargs` anymore.

Reply:
😓 definitely my bad here, somehow completely missed that line. Should be fixed in fcf8e23.